src/frontends/pytorch/src/transforms/prim_list_unpack_replacer.cpp

```diff
@@ -137,6 +137,62 @@ PrimListUnpackReplacer::PrimListUnpackReplacer() {
 
             return true;
         }
+
+        if (auto meshgrid = cast_fw_node(input_node, "aten::meshgrid")) {
+            // Input - ListConstruct
+            auto meshgrid_input_node =
+                cast_fw_node(meshgrid->input_value(0).get_node_shared_ptr(), "prim::ListConstruct");
+            if (!meshgrid_input_node) {
+                return false;
+            }
+            NodeVector rt_copy_from{list_unpack, input_node, meshgrid_input_node};
+            OutputVector meshgrid_inputs;
+            for (auto& input : meshgrid_input_node->inputs()) {
+                meshgrid_inputs.push_back(input.get_source_output());
+            }
+
+            auto meshgrid_attrs = meshgrid->get_attrs();
+            std::string indexing = meshgrid_attrs["indexing"];
```

Comment:
```suggestion
            std::string indexing = meshgrid_attrs.at("indexing");
```
Should be safer to do it that way

Reply:
Also please check that `indexing` exist in `attrs`

Reply:
Added check if "indexing" exist in attributes, added check for indexing value correctness and implemented suggested change